api-reference/v1.0/api/groupsetting_delete.md

```diff
@@ -0,0 +1,58 @@
+# Delete a group setting
+
+Delete a group setting.
+
+### Prerequisites
+
+One of the following **scopes** is required to execute this API: *Directory.ReadWrite.All* or *Directory.AccessAsUser.All*
```

Comment:
optional: add a comment about how only tenant admins have permissions to perform the create/update/delete operations